lib/main.dart

```diff
@@ -8,6 +8,7 @@ import 'package:pull_to_refresh/pull_to_refresh.dart';
 import 'app/routes/app_pages.dart';
 import 'package:form_builder_validators/form_builder_validators.dart';
 import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:google_mobile_ads/google_mobile_ads.dart';
 
 void main() async {
   runApp(const MyApp());
```

Comment:
```suggestion
  WidgetsFlutterBinding.ensureInitialized();
  MobileAds.instance.initialize();
  runApp(const MyApp());
```
[google_mobile_ads: ^2.3.0](https://pub.dev/packages/google_mobile_ads/example)のexampleだと,
アプリを実行する前に初期化する必要がありそうなので、ここに書きましょう。

Reply:
ありがとうございます！初期化タイミングを修正しました。